### PR TITLE
Use flexbox for the navbar CSS

### PR DESCRIPTION
**Pull request by sigsiroos**
Changed css for header to use flexbox and tweaked stylings to match the twitter header accordingly

### CODE REVIEW

**Comment by sindresorhus** [comment]
This doesn't correctly handle the loading indicator:

screen shot 2018-10-02 at 14 35 53

**Comment by sindresorhus** [comment]
If I make the window smaller, the loading indicator ends up above the buttons:

screen shot 2018-10-28 at 14 09 39

**Comment by sindresorhus** [comment]
When the window is very wide, the nav buttons are too much to the left. Would be nice to move them closer to the center:

screen shot 2018-10-28 at 14 10 59

**Comment by sigsiroos** [comment]
Yikes sorry about that, didn't think of that and the numerous other problems my changes were causing. Fixed this and also covered the other pages being weird by me like e.g. profile and account settings page. However, I get this problem with xo giving errors, something related to eslint or xo itself?

**Comment by sindresorhus** [comment]
I still think the nav buttons are too much to the left when the window is wide, like showin in https://github.com/sindresorhus/refined-twitter/pull/144#issuecomment-433682110 They could be much closer to the search input.

**Comment by sindresorhus** [comment]
> However, I get this problem with xo giving errors, something related to eslint or xo itself?

This is fixed in master branch. Just rebase from master.

File: source/features/inline-code.js [hunk 1]
```diff
@@ -34,9 +34,10 @@ export default function () {
 
 		const frag = text.map(val => {
 			// Style the single backticks while ignoring the already styled multiline code blocks
-			if (isElement(val)) {
+			if (isElement(val) || typeof val === "object") {
```
Comment (sindresorhus) [inline source/features/inline-code.js]: Can you add a comment about this change? Not clear why it's needed. It should also use single-quotes.

Comment (sigsiroos) [reply source/features/inline-code.js]: Oh sorry, I have commited a suggestion on pulling the navbar closer to the center but rebasing onto master didn't solve it. What am I doing wrong?

image
Don't mind the async error, this one is resolved ofc.

Regarding the change on inline-code.js, I removed that check. Previously for some reason the text array would contain elements that were objects and split would thus throw us an error but it seems to have disappeared now so it is removed

**Comment by sindresorhus** [comment]
@sigsiroos Looks like you had a bad rebase. Lots of unrelated commits included in the PR now.

**Comment by sindresorhus** [comment]
Closing for lack of activity. Happy to land a proper fix if you ever get to finish this.

**Pull request closed without merging**